src/components/Button/Button.scss

```diff
@@ -78,3 +78,11 @@ $button-lod-hover-color:            $color-ui-grey-dark;
   display: block;
   width:   100%;
 }
+
+@each $color-name, $color-value in $colors {
+  .#{'c-button--' + $color-name} {
+    background-color: #{$color-value} !important;
```

Comment:
`!important` should only ever be used in utility classes, where they're intended to be a sledgehammer to make a style happen. Components should rely on the cascade to enable their styling.